docs/containers-storage.conf.5.md

```diff
@@ -30,9 +30,7 @@ The `storage` table supports the following options:
 
 **driver**=""
   container storage driver (default: "overlay")
-  Default Copy On Write (COW) container storage driver
-  Valid drivers are "overlay", "vfs", "devmapper", "aufs", "btrfs", and "zfs"
-  Some drivers (for example, "zfs", "btrfs", and "aufs") may not work if your kernel lacks support for the filesystem
+  Default Copy On Write (COW) container storage driver. Valid drivers are "overlay", "vfs", "devmapper", "aufs", "btrfs", and "zfs". Some drivers (for example, "zfs", "btrfs", and "aufs") may not work if your kernel lacks support for the filesystem
```

Comment:
missing end period

Reply:
There are a lot of missed periods.  Fixed.